src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,6 +17,12 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
+    use_bluestore=false
```

Comment:
@tchaikov 
This the left-over of a rather old PR.
So I'll have to check it. I saw more or less the same when redoing this PR.
But I think I remember that blue_store was actually the trigger for this all.
Got to check what happens when I do run these tests...